package.json

```diff
@@ -42,6 +43,7 @@
     "accessible-autocomplete": "^2.0.3",
     "applicationinsights": "^1.8.8",
     "awilix": "^5.0.0",
+    "axios": "^0.21.1",
```

Comment:
might as well try to bump axios to version 0.21.4 and confirm all is working fine.
There is dependabot PR referring to this update with 97% compatibility.
There doesn't seem to be any major changes or breaking changes, however we should still check that we don't have any edge cases
https://github.com/hmcts/pip-frontend/pull/67